config/database.toml.example

```diff
@@ -0,0 +1,13 @@
+[development]
+username = "<username>"
+password = "<password>"
+host = "localhost"
+port = 3306
+database = "BacterialGrowth"
+
+[test]
+username = "<username>"
+password = "<passwod>"
+host = "localhost"
+port = 3306
+database = "BacterialGrowthTest"
```

Comment:
The section heading is the environment, which may also have a "production" section if we want to, say, pop a db console in the live environment for a quick query. Though it would need to have some limitations to prevent accidental issues.

This is just an example file, so 1) we don't commit passwords to the repo, but 2) have a template to copy when someone initializes the repo.

I prefer a centralized config file that is independent from streamlit to avoid coupling the db layer to the GUI and so we don't have to rely on streamlit's specific naming conventions or config location.